docs/SystemDesign/DataQuery/QueryFundamentals.md

```diff
@@ -34,12 +34,12 @@ IoTDB stores sequential and un-sequential TsFiles separately under `data/sequenc
 
 It should be noted that, in the following query documents, we tend to use `seq file` to represent Sequential files for short. For un-sequential ones we use `unseq file`. Sometimes `unordered file` or `out-of-order file` are also used as aliases of un-sequential files.
 
-## General query process
+## General process of reading TsFile
```

Comment:
This part is not only introducing reading TsFile from disk, it also contains the process of reading memory data, which is not in TsFile yet.